### DISCUSSION TITLE
apache and meshcentral

### DISCUSSION THREAD

**Discussion by Nothing2Do** [Ylianst/MeshCentral#5311] [Q&A]
I wonder if it's possible to host a meshcentral server and an apache on the same server (with "namebased" server, perhaps) ? I know how to do it apache side (virtualhost, as here : https://httpd.apache.org/docs/2.4/en/vhosts/name-based.html) but node side is unknown to me ...

**Comment by si458** [comment]
This isn't possible even with standard linux, as you can't run 2 applications on the same port!
You would have to use apache as a reverse proxy on port 80, meshcentral on say port 81 then let apache do the virtual host names and forward stuff to port 81 for meshcentral

**Comment by bbrendon** [comment]
Multiple IPs on one server.
